cli/src/pcluster/models/imagebuilder.py

```diff
@@ -80,6 +80,28 @@ def version(self):
         """Return the version of ParallelCluster used to create the stack."""
         return self._get_tag("pcluster_version")
 
+    @property
+    def image(self):
+        """Return created image by imagebuilder stack."""
+        try:
+            image_id = self.image_id
+            if image_id:
+                image_info = AWSApi.instance().ec2.describe_image(image_id)
+                return ImageInfo(
```

Comment:
You don't need to pass a restricted dictionary to ImageInfo, you can pass the full one, it will be stored in _image_data and then you already have the properties in ImageInfo to return the right attributees.

The goal of having ImageInfo class is that you don't need to do get("Name"), get("Description") etc everywhere.

You can even think to modify the describe_image method in ec2_client to directly return an ImageInfo object.

Reply:
correct, I will modify the method `describe_image` in ec2_client later with a separate commit to make it clear